recipes/containerpilot/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "containerpilot" %}
+{% set version = "2.7.8" %}
+{% set sha1= "09158be44c3e887244581d4d019748df9fcfa93c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/joyent/{{ name }}/releases/download/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha1: {{ sha1 }}
+
+build:
+  number: 0
+  script:
+    - cp {{ name }} $PREFIX/bin
+  skip: True  # [ not linux ]
+
+requirements:
+  build:
+  run:
+
+test:
+  commands:
+    - containerpilot -version
+
+about:
+  home: https://www.joyent.com/containerpilot
+  license: MPL-2.0
+  license_family: OTHER
```

Comment:
- MPL is not a valid license family, so I am using OTHER.
  - The license file is not included in the tarball

Reply:
Under the terms of the license, the source code must be included along with the executable:

```
such Covered Software must also be made available in Source Code Form, as described in Section 3.1, and You must inform recipients of the Executable Form how they can obtain a copy of such Source Code Form by reasonable means in a timely manner, at a charge no more than the cost of distribution to the recipient; and
```

Reply:
Okay, but it isn't... so how should we proceed? Keep two things in mind:

 - "go" source code with vendor dependencies is usually pretty big
 - the dev-url is pointing to the GitHub page, so they can obtain the source code without any cost.

Reply:
Actually, negate this.  "made available" does not mean "distribute".  The dev-url should be enough.